packages/cashier/src/components/crypto-fiat-converter/crypto-fiat-converter.tsx

```diff
@@ -107,7 +77,10 @@ const CryptoFiatConverter = observer(
                                 setArrowIconDirection('right');
                             }}
                             onChange={(e: TReactChangeEvent) => {
-                                onChangeConverterFromAmount(e, from_currency, to_currency);
+                                const from_rate = getRate(from_currency || '');
+                                const to_rate = getRate(to_currency || '');
+                                const exchanged_rate = (Number(e.target.value) * to_rate) / from_rate;
```

Comment:
```suggestion
                                const exchange_rate = (Number(e.target.value) * to_rate) / from_rate;
```

Reply:
Can we make this change all over this PR?

Reply:
sure. will rename them all

Reply:
@balakrishna-deriv I think `converted_amount_ is more meaningful. what do you think? :thinking:

Reply:
`converted_amount` sounds good :D